setup.py

```diff
@@ -60,8 +60,8 @@ def bleach_tokens(tokens):
     # TODO __await__, ...?
     used_space = None
     for space, toknum, tokval in tokens:
-        if toknum in [ASYNC, AWAIT]:  # TODO Python 3.7+
-            # When remove async or await, we want to use the whitespace that
+        if tokval in ["async", "await"]: # TODO Python 3.7+
```

Comment:
Sorry if I was not clear. You need to remove the "# TODO Python 3.7+" comment, but not the following lines.